### ISSUE TITLE
How do we change the settings of an IRC-service room?

### ISSUE THREAD

**Issue by bluddy** [vector-im/vector-web#1736]
We have an IRC room #freenode_#ocaml:matrix.org that we'd like to direct people to for ease-of-use. However, the room doesn't allow for guests currently, and the admin is stated to be @appservice-irc:matrix.org.

Is there any way we can gain admin rights to the room and modify its settings?

**Comment by ara4n** [comment]
@bluddy - sorry for missing this. we are currently rolling out better ways of setting up bridges for rooms so that folks can better control who has ops in the room. it should land during August; sorry for the delay.  If this is super-urgent we can special case #freenode_#ocaml to give someone ops there in the short term.

**Comment by bluddy** [comment]
@ara4n do those tools now exist?

**Comment by ara4n** [comment]
yes, they do, although it's pretty basic: matrix-org/matrix-appservice-irc#206

**Comment by t3chguy** [comment]
ping
